sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -3732,43 +3852,24 @@
                     ],
                     "content": "{{typeof data.assessment?.livingAllowance === 'number' ? '$' + (data.assessment.livingAllowance).toFixed(2) : '-'}}",
```

Comment:
not part of this PR, we do have javascript builts functions to check number like `Number.isInteger`

Reply:
The reason why we have typeOf and not Number.isInteger is Number.isInteger(2.03) return false and typeof 3.14                   // Returns "number". In the assessment json we will have values with decimals.

Reply:
The purpose of using this logic is, to differentiate between 0 and undefined. Yes Number.isInteger can be used only if we get integers from the service. We might get non integers as well like 115.50. We even tried Number.isNan it was returning false for undefined surprisingly.

Reply:
mm, okay, i got it @dheepak-aot

Reply:
@ann-aot I talked with @guru-aot and we tested with isNaN. It seem to be better. He is going to change some code and share to see how it looks like.

Reply:
mm, actually, i also did some couple testing, here are the results, 
![image](https://user-images.githubusercontent.com/77353155/152040376-00bf0573-3a03-4e69-91b0-834e7b298c38.png)

Reply:
For an example scenario
{{(isNaN(data.assessment?.transportationCost) ? '-' : '$' + (data.assessment.transportationCost).toFixed(2))}}

when transportationCost is null, then it will throw "null is not an object (evaluating '(data.assessment.transportationCost).toFixed')."

Not changing as typeOf works for most of the scenarios